### PR TITLE
bump major also

### PR DESCRIPTION
**Pull request by sbe-arg**
Bump major to facilitate

      - name: tag
        uses: anothrNick/github-tag-action@v1

as well of

      - name: tag
        uses: anothrNick/github-tag-action@v1.40.0

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+        with:
+          fetch-depth: '0'
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        id: git_auto_commit
+        with:
+          commit_message: Bump version
+
+      - name: version-tag
+        uses: anothrNick/github-tag-action@1.40.0 # another pr require to move this to v1
```
Comment (sammcj) [inline .github/workflows/main.yml:22]: 1.42.0

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,16 +1,39 @@
...
+        if: steps.git_auto_commit.outputs.changes_detected == 'true'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: version-tag-major
```
Comment (sammcj) [inline .github/workflows/main.yml:27]: I'm not sure I understand what this is doing or what value it has.

- @v1 will always use the latest v1.x.x tag.
- The below script should have a comment describing what it's doing and why we can't use this very action to achieve it.

Comment (sbe-arg) [reply .github/workflows/main.yml:27]: Here https://github.com/anothrNick/github-tag-action/tags everytime we create a new release we sould also update the `major` tag to reflec the change of the latest minor/patch.

So when consumer want to refer to @1 can do it in the same way as @1.42.0

if you use tools like https://github.com/sethvargo/ratchet then using @1 is convinient same as with checkout@v3 and not checkout@v3.0.2

Comment (sbe-arg) [reply .github/workflows/main.yml:27]: Is a different purpose with this very action this action bumps major, does not updates existing tags to update the reference

Comment (sbe-arg) [reply .github/workflows/main.yml:27]: image
checkout example

Comment (sbe-arg) [reply .github/workflows/main.yml:27]: I did the same here https://github.com/bridgecrewio/checkov-action/pull/92

Comment (sammcj) [reply .github/workflows/main.yml:27]: Ohhhh I've got you! Understood.

**Review by sammcj — APPROVED** [review]

**Pull request merged**